pkg/chain/celo/ids.go

```diff
@@ -0,0 +1,64 @@
+//+build celo
+
+package celo
+
+import (
+	cecdsa "crypto/ecdsa"
+	"fmt"
+
+	"github.com/celo-org/celo-blockchain/common"
+	"github.com/celo-org/celo-blockchain/crypto"
+
+	"github.com/keep-network/keep-ecdsa/pkg/chain"
+)
+
+func (cc *celoChain) UnmarshalID(idString string) (chain.ID, error) {
+	if !common.IsHexAddress(idString) {
+		return nil, fmt.Errorf(
+			"[%v] is not a valid ethereum ID",
+			idString,
+		)
+	}
+
+	return celoChainID(common.HexToAddress(idString)), nil
+}
+
+func (cc *celoChain) PublicKeyToOperatorID(publicKey *cecdsa.PublicKey) chain.ID {
+	return celoChainID(crypto.PubkeyToAddress(*publicKey))
+}
+
+// celoChainID is the local chain.ID type; it is an alias for
+// go-ethereum/common.Address.
+type celoChainID common.Address
+
+func (ci celoChainID) ChainName() string {
+	return "local"
```

Comment:
I am confused, why `"local"` here?

Reply:
… Listen, just 'cause I'm copying and pasting doesn't mean you get to call me out on it :laughing:

I was hoping to find a good way to introduce some sort of common `ids` package but sadly I had to resort to copy/paste. Will fix.